person.rb

```diff
@@ -1,9 +1,16 @@
 # Class that represents a person
-class Person
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'correct_name method must be implemented in a subclass.'
+  end
+end
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super
```

Comment:
- [ ] An `ArgumentError` occurs in the `Person` class when I try to run the `person.rb` file as you can see in the attached image below 👇 Please try to fix it 
![image](https://user-images.githubusercontent.com/46839548/216207155-b0678546-90fb-40c9-ba47-5d6926698d65.png)